app/main.py

```diff
@@ -1,22 +1,70 @@
 import typing as tp
 import logging
+import settings
 
 from flask import Flask, jsonify, Response
 
-import settings
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
 
 logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
 
-app = Flask(__name__, instance_relative_config=True)
-app.config.from_mapping(
-    SECRET_KEY=settings.SECRET_KEY
-)
 
+def register_flask_application(config: tp.Any) -> Flask:
+    """
+    Register flask application
+    Returns:
+        current flask app
+    """
+
+    from account.views import auth_urls, account_urls
+
+    logging.info('Starting register application')
+    app: Flask = Flask(__name__, instance_relative_config=True)
+    app.config.from_mapping(**config)
+
+    app.register_blueprint(auth_urls)
+    app.register_blueprint(account_urls)
+
+    @app.route('/')
+    def index() -> tp.Tuple[Response, int]:
+        return jsonify({'ok': 'Main page'}), 200
+
+    CORS(app)
+
+    return app
 
-@app.route('/')
-def index() -> tp.Tuple[Response, int]:
-    return jsonify({'ok': 'Main page'}), 200
 
+def register_db(application: Flask) -> SQLAlchemy:
+    """
+    Register db for appliciation
+    Returns:
+        current db session
+    """
+
+    db: SQLAlchemy = SQLAlchemy()
+    Migrate(application, db, directory='migrations')
+    return db
+
+
+def models_initialization() -> None:
+    """
+    Initialize models
+    Args:
+        db: current db state
+    """
+
+    logging.info('Starting initialization of models')
+    app.app_context().push()
+    db.init_app(app)
+    db.create_all()
```

Comment:
this should be done by alembic separately from application startup (this is a part of deploy)

Reply:
removed